templates/threatconnect.hbs

```diff
@@ -1461,6 +1463,170 @@
                       </div>
                     </div>
                   </div>
+
+                  <div class="d-flex align-items-center justify-content-between">
+                    <div class="p-title" style="margin-left: 5px;">
+                      Notes
+                    </div>
+                  </div>
+
+                  <div class="note-container">
+                    {{#unless (gt (get case.notes.data "length") 0)}}
+                      <div class="p-footnote mb-2">
+                        <span>No Notes</span>
+                      </div>
+                    {{/unless}}
+
+                    {{! Add Integration Data Notes }}
```

Comment:
Currently we do not allow adding integration data to a case if the "Enable Adding Notes to a Case" option is not set to `true`.  We need to decide if we want to allow users to add integration data to a case without the ability to add notes.  If we do, we need separate options for controlling this behavior.  In either case, if we are not allowing the user to add notes to the case, we should not show the UI for adding integration data.